crates/bevy_asset/src/server/mod.rs

```diff
@@ -775,12 +811,20 @@ impl AssetServer {
         self.data.infos.read().contains_key(id.into())
     }
 
+    /// Returns an active untyped asset id for the given path, if the asset at the given path has already started loading,
+    /// or is still "alive".
```

Comment:
```suggestion
    /// or is still "alive".
    ///
    /// Equivalently, returns [`None`] if the asset has not yet started loading, or the asset was loaded but all strong handles were dropped.
```
Is this true? I had trouble understanding what "still alive" meant.

Reply:
I looked through the asset server code. Unless I missed something, "Still alive" just means that it has a strong handle somewhere. An asset's `UntypedAssetId` is inserted into the asset server's `HashMap` of `AssetPath`s to `UntypedAssetId`s when a handle for it is created (which happens very soon after it starts loading), and it is removed when all strong handles are dropped.

Reply:
@thepackett That's my reading of it too. The comment I have there is just a copy/paste of another method's documentation with "handle" replaced with "id".